src/charts/LineChart.js

```diff
@@ -238,6 +252,32 @@ function LineChart( properties ) {
     };
   };
 
+  this.appendTable = function() {
+    var table = d3.select( this.tableSelector ).append( 'table' );
+
+    table.classed( getObjProp( 'table', this.classes ), true );
+
+    table.append( 'thead').html( getObjProp( 'thead', this.content ) );
```

Comment:
Add an `if` statement so that `thead` is only generated if the user provides content for it.